### ISSUE TITLE
Black screen, only sound works

### ISSUE THREAD

**Issue by mcpalmer1980** [RobertBeckebans/RBDOOM-3-BFG#445]
My system is an Optimus Laptop with Intel and GeForce MX150. On Linux Mint 18.3 the game would work on nVidia but only give only a black screen on Intel. The sound worked fine either way. I've been running Ubuntu 19.10 for about two days and I get a black screen on both cards.

I installed from my repo and am running in OpenGL. Unsure if game supports Vulcan and I don't think it's installed on my system. After deleting the cfg file from my old installation the game wills tart in a window and show the game select screen, but it turns black after selecting d3bfg.

```
QA Timing INIT: 000984ms
RBDOOM 3 BFG 1.1.0.1401 linux-amd64
found interface lo - loopback
found interface wlp3s0 - 192.168.254.12/255.255.255.0
no 'base' directory in exe path /usr/games, skipping
no 'base' directory in cwd path /home/michael, skipping
WARNING: using hardcoded default base path /usr/share/games/doom3bfg
------ Initializing File System ------
Current search path:
/home/michael/.rbdoom3bfg/base
/usr/share/games/doom3bfg/base
file system initialized.
--------------------------------------
Sys_Error: Couldn't load default.cfg
michael@mcpalmer-Acer:~$ rbdoom3bfg
QA Timing INIT: 000168ms
RBDOOM 3 BFG 1.1.0.1401 linux-amd64
found interface lo - loopback
found interface wlp3s0 - 192.168.254.12/255.255.255.0
no 'base' directory in exe path /usr/games, skipping
no 'base' directory in cwd path /home/michael, skipping
WARNING: using hardcoded default base path /usr/share/games/doom3bfg
------ Initializing File System ------
Loaded resource file ...

Current search path:
/home/michael/.rbdoom3bfg/base
/usr/share/games/doom3bfg/base
--------------------------------------
WARNING: Unable to open resource file maps/_startup.resources
----- Initializing Decls -----
------------------------------
Couldn't open journal files
/proc/cpuinfo CPU processors: 4
/proc/cpuinfo CPU logical cores: 8
execing default.cfg
execing D3BFGConfig.cfg
couldn't exec autoexec.cfg
----- R_InitOpenGL -----
Initializing OpenGL subsystem
Using 8 color bits, 24 depth, 8 stencil display
Using GLEW 2.1.0
GameController 0 name: PS4 Controller
GameController 0 is mapped as "030000004c050000cc09000011810000,PS4 Controller,a:b0,b:b1,back:b8,dpdown:h0.4,dpleft:h0.8,dpright:h0.2,dpup:h0.1,guide:b10,leftshoulder:b4,leftstick:b11,lefttrigger:a2,leftx:a0,lefty:a1,rightshoulder:b5,rightstick:b12,righttrigger:a5,rightx:a3,righty:a4,start:b9,x:b3,y:b2,".
OpenGL Version   : 4.6
OpenGL Vendor    : NVIDIA Corporation
OpenGL Renderer  : GeForce MX150/PCIe/SSE2
OpenGL GLSL      : 4.6
idCommon::VPrintf: truncated to 4094 characters
OpenGL Extensions: ...

   maxTextureAnisotropy: 16.000000
...using GL_EXT_texture_lod_bias
X..GL_GREMEDY_string_marker not found
...using GL_ARB_framebuffer_object
...using GL_EXT_framebuffer_blit
----- Initializing Render Shaders -----
----- Initializing Sound System ------
Setup OpenAL device and context... Done.
OpenAL vendor: OpenAL Community
OpenAL renderer: OpenAL Soft
OpenAL version: 1.1 ALSOFT 1.19.1
OpenAL extensions: ...

sound system initialized.
--------------------------------------
------- Initializing renderSystem --------
Binarize File: '_default' - reason 'generated image'
Binarize File: '_white' - reason 'generated image'
Binarize File: '_black' - reason 'generated image'
Binarize File: '_flat' - reason 'generated image'
Binarize File: '_alphaNotch' - reason 'generated image'
Binarize File: '_fog' - reason 'generated image'
Binarize File: '_fogEnter' - reason 'generated image'
Binarize File: '_noFalloff' - reason 'generated image'
Binarize File: '_quadratic' - reason 'generated image'
Binarize File: '_jitter1' - reason 'generated image'
Binarize File: '_jitter4' - reason 'generated image'
Binarize File: '_jitter16' - reason 'generated image'
Binarize File: '_random256' - reason 'generated image'
Binarize File: '_heatmap5' - reason 'generated image'
Binarize File: '_heatmap7' - reason 'generated image'
Binarize File: '_grain1' - reason 'generated image'
Binarize File: '_smaaInput' - reason 'generated image'
Binarize File: '_smaaArea' - reason 'generated image'
Binarize File: '_smaaSearch' - reason 'generated image'
Binarize File: '_scratch' - reason 'generated image'
Binarize File: '_scratch2' - reason 'generated image'
Binarize File: '_accum' - reason 'generated image'
Binarize File: '_currentRender' - reason 'generated image'
Updating texture filter parameters.
renderSystem initialized.
--------------------------------------
Skipping Intro Videos!
Reading strings/english.lang as UTF-8
9097 strings read
--------- Initializing Game ----------
gamename: baseDOOM-1
Initializing event system
...535 event definitions
Initializing class hierarchy
...158 classes, 992960 bytes for event callbacks
Initializing scripts
/proc/cpuinfo CPU frequency: 3382.67 MHz
Compiled '/usr/share/games/doom3bfg/base/script/doom_main.script': 288.7 ms
---------- Compile stats ----------

Memory usage:
     Strings: 120, 15840 bytes
  Statements: 95386, 3815440 bytes
   Functions: 3070, 496728 bytes
   Variables: 239712 bytes
    Mem used: 6284512 bytes
 Static data: 6266520 bytes
   Allocated: 8567312 bytes
 Thread size: 14064 bytes

...6 aas types
game initialized.
--------------------------------------
Loaded BinkDec file: 'video/intro/introloop.bik', looping=1512x512, 30.000000 FPS, 10800.000000 sec
Loaded BinkDec file: 'video/mars1.bik', looping=0512x512, 30.000000 FPS, 28950.000000 sec
Loaded BinkDec file: 'video/mars_rotation.bik', looping=11567x406, 30.000000 FPS, 90000.000000 sec
----- idRenderModelManagerLocal::EndLevelLoad -----
    0 models purged from previous level,     3 models kept.
---------------------------------------------------
----- idImageManager::EndLevelLoad -----
   30 images loaded in   0.0 seconds
----------------------------------------
----- idSoundSystemLocal::EndLevelLoad -----
    7 sounds loaded in   0.1 seconds
----------------------------------------
------------- Warnings ---------------
during DOOM 3: BFG Edition initialization...
WARNING: Unable to open resource file maps/_startup.resources
1 warnings
Preloading images...
00362 images preloaded ( or were already loaded ) in   0.2 seconds
----------------------------------------
00259 sounds preloaded in   0.1 seconds
----------------------------------------
WARNING: Couldn't load image: _doomclassic : _doomclassic#__0200
--- Common Initialization Complete ---
QA Timing IIS: 005907ms
WARNING: terminal type 'xterm-256color' is unknown. terminal support may not work correctly
terminal support enabled ( use +set in_tty 0 to disabled )
pid: 10673
WARNING: unknown event 1619
WARNING: unknown event 770
Added local user: mcpalmer-Acer
Opening IP socket: localhost:27015
execing joy_righty.cfg
execing joy_360_0.cfg
```

**Comment by mcpalmer1980** [comment]
My black screen problem seems to have been caused by accidentally activating VR mode. After reinstalling Doom 3 in Wine and executing the game it worked fine. Then I went to the 3D option menu and the screen soon went blank. Running the game after that always resulted in a black screen.

This is probably what happened the first time my black screen problem began as well. Curiosity brought me to the 3D options menu and an arguably poor menu design led me to accidentally activate VR mode, which blanked the screen. Perhaps 3D options should be more appropriately named VR options?

Deleting the cfg file did not fix this problem, but I eventually discovered that deleting the savegame files does. Apparently options are also saved there and then loaded from there on startup.
